rfc/rfc021-vp_token-grant-type.md

```diff
@@ -122,6 +122,10 @@ The second paragraph describes the requirements that apply to JWT encoded VP's.
 5. The `kid` header MUST be a DID URL and MUST resolve to a verificationMethod in the DID Document. The DID part of the DID URL MUST match the `iss` field.
 6. The `sub` field MUST match the `credentialSubject.id` field from all the Verifiable Credentials that are used to request the access token.
 7. The `aud` field MUST match the DID of the Authorization Server.
+8. The `iat` field MUST be present and contain a valid timestamp. It MUST be before the current time.
```

Comment:
on, or before?

Reply:
when validating, before. (with clockskew it may be one as well)